lib/services/local/access_list.go

```diff
@@ -237,6 +237,11 @@ func (a *AccessListService) DeleteAllAccessLists(ctx context.Context) error {
 	return trace.Wrap(a.service.DeleteAllResources(ctx))
 }
 
+// GetSuggestedAccessLists returns a list of access lists that are suggested for a given request. This is not implemented in the local service.
+func (a *AccessListService) GetSuggestedAccessLists(ctx context.Context, accessRequestID string) ([]*accesslist.AccessList, error) {
+	return nil, trace.NotImplemented("GetSuggestedAccessLists should not be called")
```

Comment:
I'm not sure what `services/local` is, can I return `NotImplemented` here? I had to add this method because of `services/access_list`.

Reply:
This is reasonable. This is the backend implementation, so this is more or less ensuring we can only call this function through our gRPC service.